cli/src/main.rs

```diff
@@ -10,7 +10,7 @@ use common::errors::StringErrorResult;
 use gumdrop::Options;
 use server;
 use std::env::current_exe;
-use std::io::{self, Read};
+use std::io::{self, BufRead};
```

Comment:
how does it help to use an internal buffer to read the whole content of stdin?

Reply:
iterator over the lines of an instance of BufRead we need this trait is in scope